core-java/src/main/java/com/baeldung/tostring/CustomerS1.java

```diff
@@ -0,0 +1,32 @@
+package com.baeldung.tostring;
+
+public class CustomerS1  {
+	private String firstName;
```

Comment:
We have an Eclipse formatter that you can find in our author documentation that we ask all of our authors run their code through prior to creating a PR. Can you apply it to your code changes? It will fix up random spaces like this one.